src/index.jsx

```diff
@@ -94,6 +95,7 @@ ReactDOM.render(
       <Route path='/' component={Layout}>
         <IndexRoute name='home' component={Home} {...generateRouteHooks(boundRouteActions('home'))}/>
         <Route path='about' component={About} onEnter={enterStaticPage('About')}/>
+        <Route path='about/terms' component={Terms} onEnter={enterStaticPage('Terms')}/>
```

Comment:
It's better to convert this into nested structure: "about" the top-level without components, IndexRoute and "terms" inside of it